source/urban_cooling_model.rst

```diff
@@ -159,104 +158,103 @@ For each pixel, the model computes the estimated loss in productivity, in %, for
+    * Kc: Required. Crop coefficient, a value between 0 and 1 (see Allen et al. 1998).
+    * Albedo: A value between 0 and 1, representing the proportion of solar radiation directly reflected by the LULC class. Required if using the weighted factor approach to CC calculations.
+    * Green_area: Required. A value of either 0 or 1, 1 meaning that the LULC class qualifies as a green area (green areas >2ha have an additional cooling effect), and 0 meaning that the class is not counted as a green area.
+    * Building_intensity: A floating-point value between 0 and 1. This is calculated by dividing the floor area by the land area, normalized between 0 and 1. Required if using the weighted factor approach to CC calculations.
 
-* Reference evapotranspiration: a raster representing reference evapotranspiration (units of millimeters) for the period of interest (could be a specific date or monthly values can be used as a proxy)
+* Reference Evapotranspiration: A raster representing reference evapotranspiration (units of millimeters) for the period of interest (could be a specific date or monthly values can be used as a proxy).
 
-* Areas of interest: polygon vector delineating areas of interest (city boundaries or neighborhoods boundaries). Results will be aggregated within each shape contained in this vector
+* Area of interest: Polygon vector delineating areas of interest (city boundaries or neighborhoods boundaries). Results will be aggregated within each shape contained in this vector.
 
-* Green Area Maximum Cooling Distance (:math:`d_{cool}`) : Distance (in meters) over which large urban parks (> 2 ha) will have a cooling effect
+* Green Area Maximum Cooling Distance (:math:`d_{cool}`): Distance (in meters) over which large urban parks (>2ha) will have a cooling effect (default value: 450m).
 
-* Reference Air Temperature (:math:`T_{ref}`): Rural reference temperature (where the urban heat island effect is not observed) for the period of interest. This could be nighttime or daytime temperature, for a specific date or an average over several days. The results will be given for the same period of interest).
+* Baseline air temperature (:math:`T_{ref}`): Rural reference air temperature (where the urban heat island effect is not observed) for the period of interest. This could be nighttime or daytime temperature, for a specific date or an average over several days. The results will be given for the same period of interest.
 
-* Magnitude of the UHI Effect (:math:`UHI_{max}`) : Magnitude of the urban heat island effect, in degrees Celcius, i.e. the difference between the rural reference temperature and the maximum temperature observed in the city.
+* Magnitude of the UHI effect (:math:`UHI_{max}`): Magnitude of the UHI effect (in degC), i.e. the difference between the rural reference (baseline air) temperature and the maximum temperature observed in the city.

```

Comment:
I kind of like using the unicode degree symbol (°) where possible. It's probably fine but I wonder if "degC", especially run together as one word, might be confusing to users who aren't native English speakers. Spelling out fully and/or using SI conventions may be more universally understandable.

Reply:
I agree. I somewhat lazily choose 'degC' here this time to make it short and consistent because there were a variety of spellings including an incorrect one ("Celcius"; plus a couple "centigrade"s). I think the unicode symbol (°) would be better than 'degC' for the reasons @emlys describes. SI conventions and/or fully spelling out units would be better too (and more inclusive). Again, I could be convinced of almost any way here, as long as we establish a standard and stick to it.